src/Editor.js

```diff
@@ -257,6 +257,9 @@ const Editor = () => {
   const selectionBounds = getLayerBounds(
     doc.layers.filter(layer => selection.has(layer.id))
   );
+  const selectionBoundsFormRect =
+    selectionBounds.x1 === selectionBounds.y1 &&
+    selectionBounds.x2 === selectionBounds.y2;
```

Comment:
@Mxchaeltrxn I'm not sure if it's worthwhile checking this. It currently doesn't handle the case where `x1 === x2 && y1 === y2 && x1 !== y1 && x2 !== y2`, but it's also probably easier to check `state.selection.size > 0` (as you've done for deciding to show the selection rectangles).

Reply:
Agreed. I just added these changes